File: Resueltos_por_competidor/Carlos_Cobaleda/Word.cpp
/*   
Autor: Carlos Eduardo Cobaleda Moreno
Problema: Word
Juez online: Codeforces
Veredicto: Accepted
URL: https://codeforces.com/problemset/problem/59/A
*/
#include <iostream>
#include <string>
#include <cctype>
using namespace std;

int main() {
    int mayus, minus;
    mayus = 0; minus = 0;
        string c; cin >> c;
        for (int index = 0; index < c.length(); index++) {
            if (isupper(c[index])) {
            mayus++;
            } else {
                minus++;
            }
        }

        if (mayus > minus) {
            for (int indes = 0; indes < c.length(); indes++) {
            c[indes] = toupper(c[indes]);
            }
        } else {
            for (int indes = 0; indes < c.length(); indes++) {
            c[indes] = tolower(c[indes]);
            }
        }

        cout << c << endl;
    
return 0;    
	
}

File: Resueltos_por_competidor/Alejandra_Carvajal/word.py
#Autor: Alejandra Carvajal
#Problema: Word
#Juez Online: CodeForces
#Veredicto: Accepted
#URL: https://codeforces.com/problemset/problem/59/A

s = input()
ma = 0
mi = 0

for c in s:
    if c.isupper():
        ma = ma + 1
    elif c.islower():
        mi = mi + 1

if ma > mi:
    print(s.upper())
else:
    print(s.lower())
